Makefile

```diff
@@ -82,6 +90,8 @@ help:
 	@awk -F ':.*## ' 'NF == 2 && $$1 ~ /^[A-Za-z0-9%_-]+$$/ { printf "$(COLOUR_WHITE)%-25s$(COLOUR_NORMAL)%s\n", $$1, $$2}' $(MAKEFILE_LIST) | sort
 
 tools:
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.25.0
```

Comment:
Why aren't you using hermit for this?

Reply:
hermit doesn't know about protoc-gen-go (and many other go binaries that don't have pre-build binaries easily available).